tests/simpleKVBC/scripts/integrityTest.sh

```diff
@@ -14,19 +14,19 @@ cleanup
 
```

Comment:
I dont think that this tests is currently run by our CI.
Moreover, it doesn't validate anything.

Reply:
I use it for development purposes